package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "moment": "^2.29.4",
     "react": "^18.2.0",
```

Comment:
you are using React 18 here so you don't have to do `import React from "react";` on top of every file unless you are using something from react like hooks

https://stackoverflow.com/questions/71725865/do-i-need-to-import-react-in-every-file